lib/Models/WebMapServiceCatalogItem.ts

```diff
@@ -289,21 +308,25 @@ class GetCapabilitiesStratum extends LoadableStratum(
         ) &&
         service.Abstract !== firstDataDescription
       ) {
-        const traits = createStratumInstance(InfoSectionTraits);
-        traits.name = "Web Map Service Description";
-        traits.content = service.Abstract;
-        result.push(traits);
+        result.push(
+          createInfoSection(
+            i18next.t("models.webMapServiceCatalogItem.serviceDescription"),
+            service.Abstract
+          )
+        );
       }
 
       // Show the Access Constraints if it isn't "none" (because that's the default, and usually a lie).
       if (
         service.AccessConstraints &&
         !/^none$/i.test(service.AccessConstraints)
       ) {
-        const traits = createStratumInstance(InfoSectionTraits);
-        traits.name = "Web Map Service Access Constraints";
-        traits.content = service.AccessConstraints;
-        result.push(traits);
+        result.push(
+          createInfoSection(
+            i18next.t("models.webMapServiceCatalogItem.accessConstraints"),
+            service.AccessConstraints
+          )
+        );
       }
     }
     return result;
```

Comment:
For some reason these are displaying in reverse order:

**National Map**:  
![natmap](https://user-images.githubusercontent.com/6187649/87116550-7079f300-c2b9-11ea-9d88-a55042f3fe33.png)

**next**:  
![next](https://user-images.githubusercontent.com/6187649/87116554-74a61080-c2b9-11ea-95a7-82245a10de40.png)

Reply:
Yeah I noticed similiar. 
Ordering is either controlled by some defaults, or by passing in params.

https://github.com/TerriaJS/terriajs/blob/master/lib/ReactViews/Preview/DataPreviewSections.jsx#L32-L46

I think the main change is that what appeared in master as  "Data Description" is now called "Web Map Service Layer Description", hence why it gets shuffled back. But that's not a change that I did.

Reply:
Maybe I'll define an infoSectionOrder for the WMS item, the user can always override it with their stratum